docs/self-managed/platform-deployment/manual.md

```diff
@@ -29,7 +29,7 @@ Find an overview in the [Connectors Bundle project](https://github.com/camunda/c
 Operate, Tasklist, and Optimize use Elasticsearch as its underlying data store. Therefore you have to download and run Elasticsearch.
 
 :::note
-Camunda is compatible with Elasticsearch 8.8+ (see [supported environments](/docs/reference/supported-environments.md)) which you can [download here](https://www.elastic.co/downloads/past-releases/elasticsearch-8-8-0).
+As of the 8.4 release, Camunda is compatible with Elasticsearch 8.9+ and no longer supports older Elasticsearch versions. See [supported environments](/docs/reference/supported-environments.md).
```

Comment:
I suggest to make this a bit more generic, so that we don't have to update with every minor (and also due to the fact that this is not only about ES, but also JDK, OpenSearch, Keycloak, ...)

```suggestion
Please ensure to check compatability of [supported environments](/docs/reference/supported-environments.md) for your self-managed installation.
```

Reply:
Once release goes out this will stick with the `8.4` release -- with this do you still feel the same?